expo/tests/scheduler/test_allocate_workers.py

```diff
@@ -269,3 +270,15 @@ def test_post_form_valid_delete_allocation_w_bad_data(self):
             response,
             '<form method="POST" action="/scheduler/allocate/' +
             str(self.volunteer_opp.pk) + '"', count=1)
+
```

Comment:
There should be something similar that covers each case - it looks to me like there are 3 basic cases:
- a volunteer's assignments are edited via the volunteer change state (ie the assign volunteer form, any submit will do it if it's valid)
- a volunteer is assigned via the volunteer opportunity edit (the scheduler view when the allocation is added)
- a volunteer's obligation is deleted.

In this test it looks like only 1 case is exercised.